src/main/ui/app/route/SearchPage.jsx

```diff
@@ -51,6 +49,9 @@ export default class SearchPage extends Component {
         if (query) {
             url += "&q=" + query;
         }
+	if (tag) {
```

Comment:
Again, this also looks out of order by one space but it is just fine if you view the file itself